test/end-to-end/pages/dashboard/chooseApplication.js

```diff
@@ -1,31 +1,54 @@
 /* eslint-disable no-await-in-loop */
 'use strict';
 
-const content = require('app/resources/en/translation/dashboard');
+const dashboardEn = require('app/resources/en/translation/dashboard');
+const dashboardCy = require('app/resources/cy/translation/dashboard');
 const testConfig = require('config');
+const pageUnderTest = require('app/steps/ui/dashboard');
+const welshLink = 'Cymraeg';
 
-module.exports = async function() {
+module.exports = async function(language ='en') {
     const I = this;
-
+    const dashboardContent = language === 'en' ? dashboardEn : dashboardCy;
     await I.checkPageUrl('app/steps/ui/dashboard');
 
     // we do need to allow refreshing the page here as it takes time to populate ccd, and storing data in the ccd
     // database gives a success before is actually populated, so is async.
-    for (let i = 0; i <= 5; i++) {
-        await I.waitForText(content.header, testConfig.TestWaitForTextToAppear);
-        const result = await I.checkForText('Continue application', 10);
-        if (result === true) {
-            break;
+    if (language === 'en') {
+        for (let i = 0; i <= 5; i++) {
+            await I.waitForText(dashboardContent.header, testConfig.TestWaitForTextToAppear);
+            const result = await I.checkForText(dashboardContent.actionContinue, 5);
+            if (result === true) {
+                break;
+            }
+            await I.refreshPage();
         }
-        await I.refreshPage();
-    }
 
-    await I.see(content.tableHeaderCcdCaseId);
-    await I.see(content.tableHeaderDeceasedName);
-    await I.see(content.tableHeaderCreateDate);
-    await I.see(content.tableHeaderCaseStatus);
+        await I.see(dashboardContent.tableHeaderCcdCaseId);
+        await I.see(dashboardContent.tableHeaderDeceasedName);
+        await I.see(dashboardContent.tableHeaderCreateDate);
+        await I.see(dashboardContent.tableHeaderCaseStatus);
+        await I.navByClick(dashboardContent.actionContinue);
+
+    } else {
+        await I.amOnLoadedPage(pageUnderTest.getUrl(), language);
+        await I.wait(2);
+        const text = await I.grabTextFrom(dashboardContent.actionContinue);
+        // const englishDashBoardPage = await I.checkForText(welshLink, 10);
+        console.log('Text Name' + text);
+        if (text === 'Continue application') {
+            await I.click(welshLink);
+            console.log('Switching to Welsh Dashboard Page....');
+            await I.wait(2);
+            await I.navByClick(dashboardContent.actionContinue);
+            await I.wait(3);
+        } else {
+            console.log('Welsh Dashboard Page...');
+            await I.click(welshLink);
+            await I.wait(2);
+            await I.navByClick(dashboardContent.actionContinue);
+            await I.wait(5);
+        }
```

Comment:
I'm a little confused what the difference between these 2 paths is? apart from logging? They both click `welshLink`